frontend/packages/knative-plugin/src/components/eventing/BrokerDetailsPage.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
```

Comment:
we may not need dedicated details page here as only new tab is getting contributed and can make use of `HorizontalNavTab` plugin , WDYT?

```
{
    type: 'HorizontalNavTab',
    properties: {
      model:BrokerModel,
      page: {
        name: 'Triggers',
        href: 'triggers',
      },
      loader: () =>
        ...
    },
    flags: {
      required: [flag1],
    },
  },
```

Reply:
Updated.